src/main_app/components/CalendlyWidget/index.js

```diff
@@ -1,9 +1,18 @@
-import { useMemo } from 'react'
+import { useMemo, useCallback } from 'react'
 import { CalendlyEventListener, InlineWidget } from 'react-calendly'
+import axios from 'axios'
 
 import useMediaQuery from 'utils/use_media_query'
 
 function CalendlyWidget () {
+  const handleScheduledEvent = useCallback(async ({ data }) => {
+    const calendlyInviteeURI = data.payload.invitee.uri
+    try {
+      await axios.post('http://localhost:5001/wedevelop-site/us-central1/processCalendlyEventInvitee', { calendlyInviteeURI })
```

Comment:
Move the URL to a constant in the `src/main_app/constants.js` file: `PROCESS_CALENDLY_EVENT_INVITEE_ENDPOINT_URL` (I know, it's long, but you immediately understand what the variable contains 😄 )